src/Analysis/Engine/Test/AnalysisTest.cs

```diff
@@ -1280,8 +1286,7 @@ yield 1
 
 
         [DataRow(PythonLanguageVersion.V27)]
-        [DataRow(PythonLanguageVersion.V31)]
-        [DataRow(PythonLanguageVersion.V33)]
+        [DataRow(PythonLanguageVersion.V36)]
```

Comment:
Do we stop supporting `PythonLanguageVersion.V31` and `PythonLanguageVersion.V33`?

Reply:
I am a bit on a fence here. I think test was added back when 3.3 was new. Now 3.7 is actually the main line. I think actually we should be using latest... Using 3.3 here requires its installation and if it is not installed, the test does not run which is bad.

Reply:
We can have both 3.3 and 3.7

Reply:
OK